sphinx/writers/latex.py

```diff
@@ -1498,6 +1506,15 @@ def depart_bullet_list(self, node):
 
     def visit_enumerated_list(self, node):
         # type: (nodes.Node) -> None
+        def get_enumtype(node):
+            # type: (nodes.Node) -> unicode
+            enumtype = node.get('enumtype', 'arabic')
+            if 'alpha' in enumtype and 26 < node.get('start', 0) + len(node):
+                # fallback to arabic if alphabet counter overflows
+                enumtype = 'arabic'
+
```

Comment:
there is package [http://www.ctan.org/pkg/alphalph](alphalph) which may be used for extended alphabetical indexing. `\alphalph{\value{counter}}` uses `a..z`, then `aa, ab, ..., az, ba, ..., zz` then `aaa, aab, ...` etc (sic). Unfortunately we need `\value`, so if we use package we could define macro `\sphinxalphalph` which adds the `\value` wrapper automatically, and then use `\sphinxalphalph`rather than `\alph` same for `\Alph`.

Reply:
alphalph list is only generated by following markup. docutils doesn't support `aa.` as enum-list notation.
```
a.
b.
...
#.
#.
```

I guess alph-alph might not be used often. So it's okay to fallback arabic one to me.